desktop/apps/auctions2/templates/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withLayout } from 'desktop/components/react/main_layout'
+
+function Index (props) {
+  const {
+    Layout: { Header, Body, Footer },
+    Templates: { Meta }
+  } = props
+
+  return (
+    <div>
+      <Header>
+        <Meta />
+      </Header>
+      <Body>
+        hello!
+      </Body>
+      <Footer />
+    </div>
```

Comment:
If you dig into subapps in force, each one expresses its own header, footer, etc -- there isn't one generic wrapper that we reuse all over. Each has different attributes and plugs in, so to speak.

As this pattern is still evolving (and will become clearer) I'd like to keep going in the direction I'm going without any new features; when we get into `/auction2` territory hopefully it will make more sense. Right now I feel this is a simpler approach since we don't yet need a lot of control except in places where `block` portals occur (the point of this all), and the header, body, footer approach gets it done quick and easy without much overhead.